Client/Frontend/Login/LoginInfoViewController.swift

```diff
@@ -0,0 +1,380 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Storage
+import Shared
+import SwiftKeychainWrapper
+
+private let log = Logger.browserLogger
+
+class LoginInfoViewController: LoginAuthViewController {
+    
+    // MARK: UX
+    
+    struct UX {
+        static let informationRowHeight: CGFloat = 58
+        static let createdRowHeight: CGFloat = 66
+        static let standardItemHeight: CGFloat = 44
+    }
+    
+    // MARK: Section
+    
+    enum Section: Int, CaseIterable {
+        case information
+        case createdDate
+        case delete
+    }
+    
+    // MARK: ItemType
+    
+    enum InfoItem: Int, CaseIterable {
+        case websiteItem
+        case usernameItem
+        case passwordItem
+    }
+    
+    weak var settingsDelegate: SettingsDelegate?
+
+    // MARK: Private
+    
+    private let profile: Profile
+    private weak var websiteField: UITextField?
+    private weak var usernameField: UITextField?
+    private weak var passwordField: UITextField?
+    
+    private var loginEntry: Login {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+    private var isEditingFieldData: Bool = false {
+        didSet {
+            if isEditingFieldData != oldValue {
+                tableView.reloadData()
+            }
+        }
+    }
+    
+    private var formattedCreationDate: String {
+        let date = Date(timeIntervalSince1970: TimeInterval(loginEntry.timeCreated / 1_000_000))
+        let dateFormatter = DateFormatter().then {
+            $0.locale = .current
+            $0.dateFormat = "EEEE, MMM d, yyyy"
+        }
+
+        return dateFormatter.string(from: date)
+    }
+    
+    // MARK: Lifecycle
+
+    init(profile: Profile, loginEntry: Login) {
+        self.loginEntry = loginEntry
+        self.profile = profile
+        super.init()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationItem.do {
+            $0.title = URL(string: loginEntry.hostname)?.baseDomain ?? ""
+            $0.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .edit, target: self, action: #selector(edit))
+        }
+
+        tableView.do {
+            $0.accessibilityIdentifier = "Login Details"
```

Comment:
we can translate accessibility identifiers

Reply:
Fix at https://github.com/brave/brave-ios/pull/4124/commits/a3da71c505acb083a823eaeb087c7b1705c459d2